mixer/pkg/config/crd/store.go

```diff
@@ -204,17 +204,19 @@ func (s *Store) retryCreateCache(
 	tick := time.Tick(s.bgRetryInterval)
```

Comment:
Here should use NewTicker,  and should stop ticker.

Reply:
Done.